webapp/locales/de.json

```diff
@@ -25,16 +25,7 @@
     "shouted": "Empfohlen",
     "commented": "Kommentiert",
     "userAnonym": "Anonymus",
-    "socialMedia": "Wo sonst finde ich",
-    "network": {
```

Comment:
@mattwr18 why did you delete the translations? :thinking:

Reply:
I didn't! well, I must have worked through a merge conflict incorrectly... I'll add it back
weird thing is that trying to pull in the latest from master says it's up to date :woman_shrugging: